hw/top_earlgrey/dv/cov/pinmux_pad_attr.el

```diff
@@ -0,0 +1,2792 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//==================================================
+// This file contains the Excluded objects
+// Generated By User: root
+// Format Version: 2
+// Date: Fri Oct 28 03:49:48 2022
+// ExclMode: default
+//==================================================
```

Comment:
Exclusions LGTM. The 
```
drive_strength [1] 
drive_strength [2] 
drive_strength [3] 
keep_en 
od_en 
schmitt_en 
slew_rate 
```
attributes are not modelled on the open-source sid, and the WARL mechanism ties them off to zero.